src/compiler/checker.ts

```diff
@@ -4504,6 +4535,12 @@ namespace ts {
                 if (!popTypeResolution()) {
                     type = reportCircularityError(symbol);
                 }
+
+                // If the type is a fresh, unique symbol type and is not the type for this symbol, get its regular type
+                if (type.flags & TypeFlags.Unique && type.flags & TypeFlags.Fresh && type.symbol !== symbol) {
+                    type = (<UniqueType>type).regularType;

```

Comment:
No, the regular type must point back to the unique type in some way:

```ts
// input.ts
const x = Symbol();
const y = x;

// output.d.ts
declare const x: symbol(); // fresh type
declare const y: typeof x; // regular type
```

This way you can use `y` in place of `x`.